src/js/grapl-cdk/lib/service.ts

```diff
@@ -78,6 +78,20 @@ export class Service {
             environment.RUST_BACKTRACE = '1';
         }
 
+        const role = new iam.Role(scope, 'ExecutionRole', {
```

Comment:
should this role also get used in places where we manually define a non-Service lambda? E.g. grapl-dgraph-ttl and grapl-model-plugin-deployer?

Reply:
Hmm. Good question. I'd say not necessary if we don't expect to ever work with them directly, as we do for generators connecting to log sources in other accounts, but still nice to have and easy enough to fix, so I'll add.